src/main/java/com/iota/iri/TransactionValidator.java

```diff
@@ -286,6 +424,12 @@ private boolean quickSetSolid(final TransactionViewModel transactionViewModel) t
         return false;
     }
 
+    /**
+     * If the the {@code approvee} is missing request it from a neighbor.
```

Comment:
```suggestion
     * If the the {@code approvee} is missing, request it from the node's neighbor.
```

Reply:
```suggestion
     * If the the {@code approvee} is missing, request it from a neighbor.
```

Node may have more than one neighbor